contrib/kind.sh

```diff
@@ -531,12 +562,34 @@ set_default_params() {
   JOIN_SUBNET_IPV4=${JOIN_SUBNET_IPV4:-100.64.0.0/16}
   JOIN_SUBNET_IPV6=${JOIN_SUBNET_IPV6:-fd98::/64}
   KIND_NUM_MASTER=1
+  OVN_INTERCONNECT_ENABLE=${OVN_INTERCONNECT_ENABLE:-false}
+  KIND_NUM_NODES_PER_ZONE=${KIND_NUM_NODES_PER_ZONE:-2}
+  OVN_ENABLE_LOCAL_ZONE=${OVN_ENABLE_LOCAL_ZONE:-false}
   if [ "$OVN_HA" == true ]; then
     KIND_NUM_MASTER=3
-    KIND_NUM_WORKER=${KIND_NUM_WORKER:-0}
+    NUM_WORKER=${KIND_NUM_WORKER:-0}
+    # Disabled interconnect if HA is enabled (for now)
+    OVN_INTERCONNECT_ENABLE="false"
   else
-    KIND_NUM_WORKER=${KIND_NUM_WORKER:-2}
+    NUM_WORKER=${KIND_NUM_WORKER:-2}
   fi
+
+  if [ "$OVN_INTERCONNECT_ENABLE" == false ]; then
+    KIND_NUM_ADDITIONAL_ZONES="0"
+    OVN_ENABLE_LOCAL_ZONE="false"
+  else
+    KIND_NUM_ADDITIONAL_ZONES=${KIND_NUM_ADDITIONAL_ZONES:-2}
+    NUM_WORKER=${KIND_NUM_WORKER:-0}
+
+    if [ "$OVN_ENABLE_LOCAL_ZONE" == true ]; then
+      # Each node is a zone. So there can only be one node per zone.
+      KIND_NUM_NODES_PER_ZONE=1
+    fi
+  fi
+
+  KIND_NUM_WORKER=$NUM_WORKER
```

Comment:
Confusing that we set `NUM_WORKER` to `KIND_NUM_WORKER` above, and then here the opposite, `KIND_NUM_WORKER` to `NUM_WORKER`